src/NServiceBus.Core/Sagas/LoadSagaBehavior.cs

```diff
@@ -273,12 +219,18 @@ IContainSagaData CreateNewSagaEntity(SagaMetadata metadata, IInvokeHandlerContex
             return sagaEntity;
         }
 
-        IInvokeHandlerContext currentContext;
         SagaMetadataCollection sagaMetadataCollection;
 
         ISagaPersister sagaPersister;
-        ICancelDeferredMessages timeoutCancellation;
 
-        static ILog logger = LogManager.GetLogger<SagaPersistenceBehavior>();
+        static ILog logger = LogManager.GetLogger<LoadSagaBehavior>();
+
+        public class Registration : RegisterStep
+        {
+            public Registration() : base("LoadSagaBehavior", typeof(LoadSagaBehavior), "Loads a saga if any.")
+            {
+                InsertBeforeIfExists(WellKnownStep.InvokeSaga);
```

Comment:
:crying_cat_face: We need to figure out a way to register the behaviors in a given order instead (not this pull)